calyx-py/test/correctness/queues/futils/fifo.futil

```diff
@@ -0,0 +1,286 @@
+import "primitives/core.futil";
```

Comment:
Is the hope to eventually check these all in? Or is it for the purposes of latency and LUT measurement?

Reply:
The purpose was for LUT measurement, should I go ahead and delete them then?

Reply:
You'll need them for latency measurement too. Perhaps you want to check these into the other repo, where you are taking measurements?

Reply:
And then delete them from here?